### ISSUE TITLE
Sublime 3 flashes after saving

### ISSUE THREAD

**Issue by kenw232** [Monnoroch/ColorHighlighter#311]
As soon as I installed ColorHighlighter Sublime would refresh is text window after saving everytime.  I removed ColorHighlighter and the refresh/flashing stopped.  Theres a problem.

**Comment by Monnoroch** [comment]
That's because there's an on save callback that parses the whole file and highlights all the colors.
You probably have a weak hardware or very big files. You could just disable the "Highlight all" feature, then there's no such callback installed.

**Comment by Monnoroch** [comment]
I don't know about the flash though, this is weird. I could understand freezes, but I've never even seen what a "sublime flashes" is.

**Comment by kenw232** [comment]
To disable the "highlight all" do you mean drop a bunch of files from "file_exts":?  or is there something else?  I so no "highlight all" in Colorhighligher.sublime-settings.

**Comment by Monnoroch** [comment]
Nope, there's a ha_style setting, set it to "none". Alternatively, there's an option to do that in the UI.
